src/main/java/sg/ncl/MainController.java

```diff
@@ -3854,7 +3857,6 @@ private int getNumberOfRunningExperiments(int numberOfRunningExperiments, JSONAr
         SortedMap<String, Map<String, String>> globalImagesMap = new TreeMap<>();
 
         log.info("Retrieving list of global images from: {}", properties.getGlobalImages());
-
         try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)